AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -2261,7 +2261,9 @@ samlab.ws##iframe[src^="https://yoomoney.ru/"]
 !
 hromadske.ua#$#.DonateModal-overlay { display: none !important; }
 hromadske.ua#$#body { overflow: visible !important; }
-!
+inkorr.com##.sub-tg-wrapper
```

Comment:
That will block the social links in the footer. Better do another rule and add it to `annoyances.txt` - under this comment `! Annoying elements, related to social networks, which we do not block in the main Social filter`.